test/plexy/instrumentor_test.exs

```diff
@@ -15,29 +15,23 @@ defmodule Plexy.InstrumentorTest do
   end
 
   @opts Instrumentor.init([])
-  test "Instrumentor.call/2 logs the start" do
+  test "Instrumentor.call/2 logging" do
     logged =
       capture_log(fn ->
-        conn(:get, "/foobar") |> Instrumentor.call(@opts)
+        conn(:get, "/foobar")
+        |> Instrumentor.call(@opts)
+        |> Plug.Conn.send_resp(200, "wow")
       end)
 
-    [start_log_line | _finish_log_line] = String.split(logged, "\n")
+    [start_log_line, measure_log_line, finish_log_line | _empty_list] = String.split(logged, "\n")
 
     assert start_log_line =~ "instrumentation"
     assert start_log_line =~ "start"
     assert start_log_line =~ "path"
     assert start_log_line =~ "method"
-  end
-
-  test "Instrumentor.call/2 logs the finish" do
-    logged =
-      capture_log(fn ->
-        conn(:get, "/foobar")
-        |> Instrumentor.call(@opts)
-        |> Plug.Conn.send_resp(200, "wow")
-      end)
 
-    [_start_log_line, finish_log_line | _empty_list] = String.split(logged, "\n")
+    assert measure_log_line =~ "measure"
```

Comment:
what log line would the instrumentor actually produce?

Reply:
`timestamp measure#plexy.request.latency.milliseconds=0`

Reply:
also, `plexy` is the default, it will use `APP_NAME` if the env var is defined, I think this should probably be more customizable, maybe allowing it to be set in the mix config files as well? Or allowing users to specify which env var to use?

Reply:
fixed